site/en/docs/extensions/reference/commands/index.md

```diff
@@ -210,7 +204,7 @@ registration as shown in the following example.
 ```
 
 ```js
-// background.js
+// service-worker.js
```

Comment:
Can we convert this comment to a [label](https://developer.chrome.com/docs/handbook/components/#labels)? Also for the manifest file?